etc/DependencyInstaller.sh

```diff
@@ -776,6 +779,14 @@ while [ "$#" -gt 0 ]; do
             export PREFIX="${HOME}/.local"
             export isLocal="true"
             ;;
+        -constant-build-dir)
+            baseDir="/tmp/DependencyInstaller-OpenROAD"
+            if [[ -d "$baseDir" ]]; then
+                echo "INFO: removing old building directory $baseDir"
+                rm -r "$baseDir"
+            fi
+            mkdir -p "$baseDir"
```

Comment:
```suggestion
            if [[ -d "$baseDir" ]]; then
                echo "INFO: removing old building directory $baseDir"
                rm -r "$baseDir"
            fi
            baseDir="/tmp/DependencyInstaller-OpenROAD"
            mkdir -p "$baseDir"
```

Reply:
Would this be the correct order? First, remove the folder created by mktmp, then reassign the value and make the folder. Otherwise, the mktmp folder will still exist.

Reply:
Yes, I've changed the order, so now a previous `baseDir` is removed